### ISSUE TITLE
Email Alert

### ISSUE THREAD

**Issue by NotSoLateShow** [smori1983/chrome-url-notification#44]
Can the extension email me when the alert is triggered?

**Comment by smori1983** [comment]
@NotSoLateShow 

No, this extension just adds DOM element when registered pattern matches the URL of current tab.

There are 6 patterns of notification DOM element.
Below is the 'bottom' pattern.

<img width="925" alt="issue_44_1" src="https://user-images.githubusercontent.com/1549175/107136393-05bf9180-6946-11eb-9f3c-3f1415ad9dc8.png">
